consensus/reactor_test.go

```diff
@@ -239,6 +241,36 @@ func TestReactorCreatesBlockWhenEmptyBlocksFalse(t *testing.T) {
 	}, css)
 }
 
+func TestReactorReconnectionPeerGetPeerState(t *testing.T) {
+	N := 1
+	css, cleanup := randConsensusNet(N, "consensus_reactor_test", newMockTickerFunc(true), newCounter,
+		func(c *cfg.Config) {
+			c.Consensus.CreateEmptyBlocks = false
+		})
+	defer cleanup()
+	reactors, _, eventBuses := startConsensusNet(t, css, N)
+	defer stopConsensusNet(log.TestingLogger(), reactors, eventBuses)
+	reactor := reactors[0]
+	defer reactor.Stop()
+	sw := reactor.Switch
+	nodeKey := p2p.NodeKey{PrivKey: ed25519.GenPrivKey()}
+	nodeId := nodeKey.ID()
+
+	for i := 0; i < 100; i++ {
+		peer := mock.NewFixIdPeer(nil, nodeId)
+		err := p2p.AddPeerToSwitch(sw, peer)
```

Comment:
we don't need to initialize switch & add peer to it in order to test that reactor does not panic anymore. we can just call reactor.InitPeer() here and write a comment (CONTRACT) saying that if InitPeer is not called, Reactor will panic!

Reply:
I can do that